addons/binding/org.openhab.binding.sleepiq/src/main/java/org/openhab/binding/sleepiq/config/SleepIQCloudConfiguration.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.sleepiq.config;
+
+/**
+ * Configuration class for the SleepIQ cloud.
+ *
+ * @author Gregory Moyer - Initial contribution
+ */
+public class SleepIQCloudConfiguration {
+
+    public static final String USERNAME = "username";
+    public static final String PASSWORD = "password";
+    public static final String POLLING_INTERVAL = "pollingInterval";
+    public static final String LOGGING = "logging";
```

Comment:
Logging here refers to a configuration option on the API library set when the cloud connection is created. When logging is enabled, all requests and responses sent over HTTP are logged.

The static string field is not used currently, but the boolean field holding the actual configuration option is used by SleepIQCloudHandler.

```java
        logger.debug("Creating SleepIQ client");
        Configuration cloudConfig = new Configuration().withUsername(bindingConfig.username)
                .withPassword(bindingConfig.password).withLogging(bindingConfig.logging);
        cloud = SleepIQ.create(cloudConfig);
```

1. I thought that best practice was to create a static string representing each configurable property based on what other bindings were doing. I can delete the static string if you prefer. Please advise.
2. If this specific configuration option is bad practice for bindings, is there an alternative suggestion for passing debug/log options to a third party library?